walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -56,18 +63,25 @@ If you already have admin access to a Slack workspace, skip this procedure.
 
 . Enter your email address and click *Next*.
 
-. Enter a name for your workspace and click *Next*.
+. Enter the following name for your workspace and click *Next*:
++
+----
+low-code-workspace
+----
++
 The name you choose becomes part of the URL for the workspace. 
-For example, naming the workspace `fuseonline` creates the URL `fuseonlinedemo.slack.com`.
 
-. When prompted *What is your team working on?* enter `Slack Demo`. 
-This creates a *slack-demo* channel.
+. When prompted *What is your team working on?* enter:
++
+----
+Low-code Demo
```

Comment:
Maybe just `low-code-demo` to keep it simple. Spaces etc. are common issues and I've not tested this with them.